### DISCUSSION TITLE
Add Long Screenshot and Screen Recording Functions

### DISCUSSION THREAD

**Discussion by Jacen918** [flameshot-org/flameshot#3592] [Ideas]
I propose adding two highly practical function buttons to the existing features. Firstly, incorporating a "Long Screenshot" button that allows users to easily capture long page content without leaving the current interface, eliminating the cumbersome steps of frequently switching to other screenshot tools. Additionally, adding a "Screen Recording" function button, which enables users to initiate screen recording with the same quick access as the screenshot button, without having to switch to other screen recording tools, further enhancing operational efficiency and convenience.

Moreover, regarding personalization needs, I suggest allowing users to customize the icons of the function buttons. This not only makes the interface more in line with personal aesthetics but also enhances users' sense of belonging to the software. In terms of icon design, I propose introducing a transparency adjustment option, enabling users to adjust the transparency of the icon background according to their personal preferences to achieve the best visual effect. At the same time, the ability to turn off the icon shadow effect would also be a good choice, making the icons more concise and modern.

**Comment by mmahmoudian** [comment]
> Firstly, incorporating a "Long Screenshot" button that allows users to easily capture long page content without leaving the current interface, eliminating the cumbersome steps of frequently switching to other screenshot tools.

This is duplicate of #1130 (and also #3168 , #2890 , #2763 , and possibly many more)

>  adding a "Screen Recording" function button, which enables users to initiate screen recording with the same quick access as the screenshot button, without having to switch to other screen recording tools, further enhancing operational efficiency and convenience.

Duplicate of #172 (and #2201 , and possibly more)

> Moreover, regarding personalization needs, I suggest allowing users to customize the icons of the function buttons. This not only makes the interface more in line with personal aesthetics but also enhances users' sense of belonging to the software. In terms of icon design, I propose introducing a transparency adjustment option, enabling users to adjust the transparency of the icon background according to their personal preferences to achieve the best visual effect. At the same time, the ability to turn off the icon shadow effect would also be a good choice, making the icons more concise and modern.

Please convert this to feature requests, but please keep them independently. One feature request per feature. Thanks.
